Library/brew.sh

```diff
@@ -207,6 +192,9 @@ EOS
   esac
 fi
 
+source "$HOMEBREW_LIBRARY/Homebrew/utils/ruby.sh"
+setup-ruby-path
```

Comment:
That produces a desirable edge-case actually: what if a Bash command calls a Ruby command? We'll need to handle that case anyway when someone runs `brew update` which will run the old version of this shell script and then the `master` version of `brew update-report` which will require Ruby 2.0.